docs/docs/introduction/container-images.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: default
+title: Container Images
+nav_order: 2
+parent: Introduction
+---
+
+# Introduction to Containers
+
+A container is a portable encapsulation of an environment. It's a
+way for you to package an entire operating system, libraries, and
+dependencies in a package that you can move and run anywhere.
+That is the quick and dirty definition, but today we are going
+to delive a little more into what this actually means.
+
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## Container Basics
+
+### What problems do containers solve?
+
+
+#### Reproducibility
+{: .no_toc }
+
+We have differences in environments, hardware, and even security, and this makes sharing of applications challenging, leading to the commonality of the phrase above. A *reproducible* workflow that uses containers helps us get around this problem, and (hopefully) reduce the frequency of uttering this phrase _But it ran on my computer!_
+
+#### Portability
+{: .no_toc }
+
+The ability to easily move a container from one machine to another is easily afforded by the fact that many container formats are variations on a compressed filesystem. If you wanted to share your entire computer, right now, you would have a hard time doing that.
```

Comment:
I think by "move a container" you mean replicate the execution of an application with a unique configuration from a first computer to a second computer.

Reply:
I'm used to containers as single binaries, so I did in fact have in mind moving (via scp, email, some cloud storage, etc.)

Reply:
Yeah containers are normally assembled from layers. One container may reuse n-layers of another container. Sure you can create a copyable version of a container, but that would then be outside the container runtimes container image cache. When you request an instance of a container a rootfs is assembled for it from the image layers and config.json specified mounts by the container runtime. This caching and layering and mounting to create a rootfs snapshot for an instance of a run-able container image is a huge advantage for containers, vs say a monolithic uni-kernel application that everything already in it and can be copied / moved around.

Reply:
This is very different from how Singularity containers work - they are single binary (derivates of squashfs called SIF image).

Reply:
I noticed the alignment of your text with singularity..

Reply:
I can't help it, my brain is a single binary and it's read only ! :scream: